### PR TITLE
test: add basic generic list tests

### PR DESCRIPTION
**Pull request by molnosikristof**

### CODE REVIEW

**Comment by molnosikristof** [comment]
Ok, then I leave it here and talk about it. :)

File: e2e/test/list-tests/list-test.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const assert = require('assert');
+const visualRegression = require('../../utils/visual-regression');
+const { takeScreenshotAndGetWholePageCompareResult, takeScreenShotOfElement } = visualRegression;
```
[the remark this answers is not in the document]
Comment (molnosikristof) [reply e2e/test/list-tests/list-test.js]: I'm happy to make you laugh.

File: e2e/test/list-tests/list-test.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+//test names
+const DELETE_LIST_ELEMENT = 'delete-list-element';
+const ADD_NEW_ITEM_CHECK = 'add-new-item-check';
+const LIST_EDITING = 'list-editing';
+const ADD_NEW_ELEMENT = 'add-new-element';
+
+describe('FEF lists tests', () => {
+
+  it('Checks the the fef lists page title and browser compare visual regression', () => {
```
[the remark this answers is not in the document]
Comment (molnosikristof) [reply e2e/test/list-tests/list-test.js]: done

File: e2e/test/list-tests/list-test.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+const INPUT_FIELD = 'input[type="text"]';
+const SAVE_BUTTON = '.list-item-editing .button-content';
+
+//test names
+const DELETE_LIST_ELEMENT = 'delete-list-element';
+const ADD_NEW_ITEM_CHECK = 'add-new-item-check';
+const LIST_EDITING = 'list-editing';
+const ADD_NEW_ELEMENT = 'add-new-element';
+
+describe('FEF lists tests', () => {
```
[the remark this answers is not in the document]
Comment (molnosikristof) [reply e2e/test/list-tests/list-test.js]: Thanks, I will, but it is a little bit bigger change, all the reference path generated from the desc name, so can I do it in a separated PR?

File: e2e/test/list-tests/list-test.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+  it('Checks the the fef lists page title and browser compare visual regression', () => {
+    browser.url('/lists');
+    assert.equal(browser.getTitle(), 'Intellyo Application Design System');
```
[the remark this answers is not in the document]
Comment (molnosikristof) [reply e2e/test/list-tests/list-test.js]: Done

File: e2e/test/list-tests/list-test.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (ruffle1986) [reply e2e/test/list-tests/list-test.js]: Sure, it's fine with me.

**Comment by molnosikristof** [comment]
Closed. Not clear req.

**Pull request closed without merging**